src/Generator/PropertyGenerator.php

```diff
@@ -21,10 +24,38 @@ class PropertyGenerator extends AbstractMemberGenerator
 
     protected bool $isConst = false;
 
+    protected ?string $type = null;
```

Comment:
Uhhh, let's avoid having even more state :| 

Also, what is a `$type` here? Shouldn't be a `string`, but rather a `TypeGenerator`.

Reply:
You are absolutely RIGHT! Silly me, just changed it and pushed!